container/docker/handler.go

```diff
@@ -248,6 +251,7 @@ func newDockerContainerHandler(
 	handler.image = ctnr.Config.Image
 	handler.networkMode = ctnr.HostConfig.NetworkMode
 	handler.deviceID = ctnr.GraphDriver.Data["DeviceId"]
+	handler.restartCount = ctnr.RestartCount
```

Comment:
@dashpole I've tested this and works fine. The counter is increases each time a container restarts :) This is only a handler of 'how to gather this information', and it's used [here](https://github.com/estaleiro/cadvisor/blob/e42a357488c3b1cb3411b4647d61e73096230bba/container/docker/handler.go#L472)